.github/workflows/contract.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.16.7'
+          go-version: '1.17'
```

Comment:
Looking at the latest run with this version: https://github.com/snyk/snyk-iac-rules/runs/3870370450?check_suite_focus=true

It looks like it pulled 1.17.1. That surprises me, I thought https://github.com/actions/setup-go would either pull 1.17.2 from https://github.com/actions/go-versions/blob/main/versions-manifest.json, or 1.17.0 due to not specifying a semver-style constraint string.

Odd, but certainly non-blocking!